dpc-sdp/ripple-framework
# build(@dpc-sdp/nuxt-ripple-cli): :package: change postinstall script so all packages are built

**Issue**:

### What I did

- postinstall script was only building ripple packages, meaning lib packages like cli tool weren't being built before publishing to NPM
- 

### How to test

- Installing @dpc-sdp/nuxt-ripple-cli should install /dist folde in node_modules folder and bin `nuxt-ripple mock` should run.
- 

### Checklist




#### For all PR's


#### For new components only
